plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -194,3 +210,87 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
+
+	sharedIndexInformer := informerFactory.InformerFor(&clientv1.ConfigMap{}, func(client clientset.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+		sharedIndexInformer := cache.NewSharedIndexInformer(
```

Comment:
@liggitt After your comment, it seems that the informers and the way they work are very confusing. Frankly, I am not sure if I understand why this shouldn't work or if it does something that I don't want. Is there any document on how this should be done? I pretty much followed [the same thing that scheduler does to monitor nodes](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/scheduler/factory/factory.go#L194) (and pods).


@davidopp It is the case, but I added it in a previous PR to [the defaults.go file for componentconfig](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/v1alpha1/defaults.go#L172). I am not sure if it's the best way. I can change it if it's not the right approach.